python/core/auto_generated/qgssettings.sip.in

```diff
@@ -110,9 +116,12 @@ On Windows, fileName is a path in the system registry.
 
 If format is QSettings.IniFormat, fileName is the name of an INI file.
 
-Warning: This function is provided for convenience. It works well for accessing INI or .plist
-files generated by Qt, but might fail on some syntaxes found in such files originated by
-other programs. In particular, be aware of the following limitations:
+.. warning::
+
+   This function is provided for convenience. It works well for accessing INI or .plist
+   files generated by Qt, but might fail on some syntaxes found in such files originated by
+   other programs. In particular, be aware of the following limitations:
+
 - QgsSettings provides no way of reading INI "path" entries, i.e., entries with unescaped slash characters.
```

Comment:
@3nids These paragraphs should be part of the warning frame but whatever indentation I applied in the .h file they don't seem to move under